app/models/role_store.rb

```diff
@@ -6,8 +6,18 @@ def initialize
   end
 
   def build_role_store_data
-    Role.find_each do |r|
-      @data[r.name] = r.users.pluck(:username, :email).to_h
+    josh_elder_username = "jbe2215"
+    josh_elder_email  = "JoshElder@northwestern.edu"
+
+    Role.find_each do |role|
+      role_user_data = r.users.pluck(:username, :email).to_h
+
+      # update Josh's email to titlecase in the role store
+      if role_user_data[josh_elder_username]
+        role_user_data[josh_elder_username] = josh_elder_email
+      end
+
+      @data[role.name] = role_user_data
```

Comment:
Move the value of `r.users.pluck...` into a variable `role_user_data`, check if Josh's netid (username) is in the `role_user_data` and if it is update it to titlecase, store the value of `role_user_data`